.gitignore

```diff
@@ -11,4 +11,16 @@ __pycache__
 
 #IntelliJ
 .idea
-*.iml
\ No newline at end of file
+*.iml
+
+#Setup
+MANIFEST
+build/
```

Comment:
Sorry if my question is dumb, but I'm not sure if you suggested:

1. To remove `MANIFEST` and `build/` from `.gitignore` OR
2. To remove `MANIFEST.in` from filesystem